packages/conferer/test/Conferer/ConfigSpec.hs

```diff
@@ -196,4 +197,15 @@ spec = do
                   ]
                   [ ]
             res <- listSubkeys "a" c
-            res `shouldBe` ["a.k"]
\ No newline at end of file
+            res `shouldBe` ["a.k"]
+
+    describe "#addDefaults" $ do
+      context "with multiple defaults on the same key" $ do
+        it "the last one has more priority" $ do
```

Comment:
why the last one? 🤔

Reply:
No reason really, I had to make a decision so I made one that wasn't crazy and that's why I wrote the test